package.json

```diff
@@ -58,6 +58,7 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "crypto-browserify": "^3.12.0"
+    "crypto-browserify": "^3.12.0",
+    "yarn": "^1.22.19"
```

Comment:
We shouldn't include yarn as a dependency

Reply:
Oops, that was me trying to get it working on windows. Will remove